content/fast/adaptive-serving-based-on-network-quality/codelab-adaptive-serving-based-on-network-quality.md

```diff
@@ -0,0 +1,172 @@
+---
+page_type: glitch
+title: Adaptive serving based on network quality
+author: mihajlija
+description: |
+  In this codelab you’ll learn how to use the Network Information API to adapt
+  your content based on the network quality.
+web_updated_on: 2019-01-31
+web_published_on: 2019-01-31
+glitch: adaptive-serving-netinfo-starter
+---
+
+# Adaptive serving based on network quality
```

Comment:
Suggestion to retitle this "Adapt video to image serving based on network quality"